code/game/objects/items/storage/boxes.dm

```diff
@@ -1556,3 +1556,15 @@
 		/obj/item/stack/wrapping_paper/small=1
 		)
 	generate_items_inside(items_inside,src)
+
+/obj/item/storage/box/plumbing
```

Comment:
you should add this as a relatively cheap cargo pack

Reply:
exactly what i'm working on right now.